### PR TITLE
Fix glass weights, lengths and volumes

### PR DESCRIPTION
**Pull request by ANickelN**
**NEEDS TO BE MERGED AFTER #48420**

Sheets assumed to be 3' x 4'
Pane assumed to be 12" x 12"

Regular glass assumed to be 3/32" thick.
Tempered glass is written to be 1/4" thick.
Ballistic glass is assumed to be NIJ IV compliant, with a thickness of 2 1/4"

https://security-glazing.com/bullet-resistant-glazing/ (Used for ballistic glass references)

#### Summary
Bugfixes "Volume, Length and Weight of glass Corrected"

#### Purpose of change

Fixes issue #48447 and makes the other glass accurate as well.

#### Describe the solution

Added proper length, weight and volume to each type of glass in the game according to real-life values.

#### Describe alternatives you've considered

Leaving glass how it is.

#### Testing

Spawn in any of the types of glass and ensure that they are the proper weight, volume and length.

#### Additional context

God help anyone making bulletproof vehicles after this.

Monster Math:

Regular Glass is : 14.58 lbs @ 12 sq ft and 3/32" thick glass.

Tempered Glass is stated to be 6mm thick, or very close to 1/4" thick.  The volume would then be 36" x 48" x 0.25" = 432" cube
432" cube = 7080 ml

Using proportion we can see that if glass is 6577g/2655ml then tempered glass should be 17,513g/7070ml 
(Tempered Glass is noted to have essentially the same weight as standard glass)

Ballistic Glass is 36" x 48" x 2.25" = 3,888" cube
3888"cube = 63712.9 ml

12sq ft of this NIJ IV glass is stated to be 25.3lb/sq ft so.

12 x 25.3 = 303.6 lbs

### CODE REVIEW

File: data/json/items/resources/glass.json [hunk 1]
```diff
@@ -27,13 +27,14 @@
     "symbol": "]",
     "color": "light_cyan",
     "name": { "str": "sheet of glass", "str_pl": "sheets of glass" },
-    "description": "A large sheet of glass.  Easily shattered.  Useful for repairing windows.",
+    "description": "A large sheet of glass around three by four feet.  Easily shattered.  Useful for repairing windows.",
```
Comment (MarlossCDDA) [inline data/json/items/resources/glass.json:30]:
```suggestion
    "description": "A large sheet of glass, around three by four feet.  Easily shattered.  Useful for repairing windows.",
```

**Comment by Maleclypse** [comment]
Ok so in the file materials.json here is the entry for glass
```  {
    "type": "material",
    "id": "glass",
    "name": "Glass",
    "density": 14,
    "specific_heat_liquid": 0.2,
    "specific_heat_solid": 0.2,
    "latent_heat": 100,
    "bash_resist": 3,
    "cut_resist": 4,
    "bullet_resist": 3,
    "acid_resist": 10,
    "fire_resist": 3,
    "elec_resist": 4,
    "chip_resist": 0,
    "dmg_adj": [ "scratched", "cut", "cracked", "shattered" ],
    "bash_dmg_verb": "cracked",
    "cut_dmg_verb": "scratched",
    "burn_data": [
      { "fuel": 0, "smoke": 0, "burn": 0 },
      { "fuel": 0, "smoke": 0, "burn": 0 },
      { "fuel": 0, "smoke": 0, "burn": 1000, "volume_per_turn": "5000 ml", "//": "More like shattering than melting" }
    ],
    "burn_products": [ [ "glass_shard", 1 ] ]
  },```
  
  What we need is a similar entry for the ballistic glass material.  You would match the bullet resistances from the NIJ levels so that you thickness of 1 is represented.  So for example if NIJ IV standard stops a .223 round we can assume that is a bullet resistance of (guess here) 50.   And it's thickness is and a quarter would be the equivalent to 9 thickness in our json (a quarter inch thick) that's how you'd get the math for how much bullet resist each thickness should have.

**Comment by LaVeyanFiend** [comment]
> So for example if NIJ IV standard stops a .223 round we can assume that is a bullet resistance of (guess here) 50.

This is a lowball for NIJ IV, as the standard for stoppage is one .30-06 M2 AP.

> Type IV hard armor or plate inserts shall be tested in a conditioned state with .30 caliber armor piercing (AP) bullets (U.S. Military designation M2 AP) with a specified mass of 10.8 g (166 gr) and a velocity of 878 m/s ± 9.1 m/s (2880 ft/s ± 30 ft/s). ~ [NIJ Standard 0101.06](https://www.ojp.gov/pdffiles1/nij/223054.pdf)

0.5*(10.8/1000 to get kg) * (878^2) = (4162.77)^0.5 = 64.5

**Comment by actual-nh** [comment]
> God help anyone making bulletproof vehicles after this.

?

> > So for example if NIJ IV standard stops a .223 round we can assume that is a bullet resistance of (guess here) 50.
> 
> This is a lowball for NIJ IV, as the standard for stoppage is one .30-06 M2 AP.
>
> > Type IV hard armor or plate inserts shall be tested in a conditioned state with .30 caliber armor piercing (AP) bullets (U.S. Military designation M2 AP) with a specified mass of 10.8 g (166 gr) and a velocity of 878 m/s ± 9.1 m/s (2880 ft/s ± 30 ft/s). ~ [NIJ Standard 0101.06](https://www.ojp.gov/pdffiles1/nij/223054.pdf)
> 
> 0.5*(10.8/1000 to get kg) * (878^2) = (4162.77)^0.5 = 64.5

Is that stopping with no damage, or with damage to the glass?

**Comment by ANickelN** [comment]
Now that ballistic glass weighs around 300 pounds for 12sq ft sheet, it will be a lot more difficult to remove and haul around.
The glass would be damaged in that specific spot, but it should be able to take those hits to other areas as well with no issues.

**Pull request merged**